packit_service/constants.py

```diff
@@ -16,6 +16,7 @@
 CONFIG_FILE_NAME = "packit-service.yaml"
 
 TESTING_FARM_API_URL = "https://api.dev.testing-farm.io/v0.1/"
+TESTING_FARM_INSTALL_TEST_URL = "https://gitlab.com/testing-farm/tests"
```

Comment:
'install' is a verb, so can wrongly indicate this is some kind of action flag. Maybe `TESTING_FARM_INSTALLABILITY_TEST` would be better?